guide/voice/receiving-audio.md

```diff
@@ -0,0 +1,52 @@
+# Receiving Audio
+
+::: warning
+Discord does not officially support Discord Bots receiving audio ([see here](https://github.com/discordapp/discord-api-docs/issues/808)), so there may be some breakages. However, Discord.js aims to reasonably support this feature.
+:::
+
+## Basic Usage
+
+In addition to sending audio over voice connections, you can also receive audio (i.e. listen to other users and bots in a voice channel) using Discord.js.
+
+The example below will listen to a user until they stop speaking, and all the audio received from that user is decoded from Opus to signed 16-bit little-endian (s16le) PCM and stored in a file called `user_audio`.
+
+```js
+const fs = require('fs');
+
+// Create a ReadableStream of s16le PCM audio
+const audio = connection.receiver.createStream(user, { mode: 'pcm' });
+
+audio.pipe(fs.createWriteStream('user_audio'));
+```
+
+The `mode` option defaults to `'opus'` and can be specified as `'pcm'`. When `'opus'` is specified, Discord.js will not attempt to decode each received Opus packet. The stream created in this case will be a ReadableStream of Opus packets.
```

Comment:
Adding an "alternatively" in the first sentence here, i.e. "...and can alternatively be specified...", reads better.